src/sparseml/optim/manager.py

```diff
@@ -118,19 +123,70 @@ def compose_staged(
         if not isinstance(additional_recipe, BaseManager):
             additional_recipe = cls.from_yaml(additional_recipe)
 
-        if isinstance(base_recipe.modifiers, OrderedDict):
-            raise ValueError(
-                "non-staged recipes not yet supported for Manager.compose_staged "
-                "found base_recipe with non_staged modifiers"
+        base_recipe_metadata = deepcopy(base_recipe._metadata)
+        additional_recipe_metadata = deepcopy(additional_recipe._metadata)
```

Comment:
let's add an `@property` for metadata so it can be accessed by `manager.metadata` instead of as a private field. may come up elsewhere